atm.js

```diff
@@ -1,3 +1,185 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
+var balance;
+var totalBalance;
+var userAmount;
+var nBalance;
+var userAmountN;
+var overDrawAmount;
+var checkingNewTotal;
+var savingsNewTotal;
+
+  $("#checking .deposit").on("click", function(e){
+      e.preventDefault();
+
+      //sets balance equal to whatever is in the checking account and makes it a number
+      balance = $("#checking .balance").html();
+      nBalance = parseFloat(balance.split("$")[1])
+
+      //sets userAmount to whatever is in the input and makes it a  number
+      userAmount = $("#checking .input").val();
+      userAmountN = parseFloat(userAmount);
+
+      depositChecking();
+
+      cNotRed();
+
+      $("#checking .input").val("");
+  })
+
+  $("#checking .withdraw").on("click", function(e){
+      e.preventDefault();
+      // $("#checking .balance").html(totalBalance);
+      //sets balance equal to whatever is in the checking account and makes it a number
+      balance = $("#checking .balance").html();
+      nBalance = parseFloat(balance.split("$")[1]);
+
+      //sets userAmount to whatever is in the input and makes it a  number
+      userAmount = $("#checking .input").val();
+      userAmountN = parseFloat(userAmount);
+
+      withdrawChecking();
+
+      $("#checking .input").val("");
+
+      overDrawChecking();
+      makeCheckingRed();
+      ifCWithdrawAtStart();
+
+      $("#checking .input").val("");
+  })
+
+  $("#savings .deposit").on("click", function(e){
+      e.preventDefault();
+
+      //sets balance equal to whatever is in the checking account and makes it a number
+      balance = $("#savings .balance").html();
+      nBalance = parseFloat(balance.split("$")[1])
+
+      //sets userAmount to whatever is in the input and makes it a  number
+      userAmount = $("#savings .input").val();
+      userAmountN = parseFloat(userAmount);
+
+      depositSavings();
+      sNotRed();
+
+      $("#savings .input").val("");
+  })
+
+  $("#savings .withdraw").on("click", function(e){
+      e.preventDefault();
+      //sets balance equal to whatever is in the checking account and makes it a number
+      balance = $("#savings .balance").html();
+      nBalance = parseFloat(balance.split("$")[1]);
+
+      //sets userAmount to whatever is in the input and makes it a  number
+      userAmount = $("#savings .input").val();
+      userAmountN = parseFloat(userAmount);
+
+      withdrawSavings();
+
+      $("#savings .input").val("");
+
+    makeSavingsRed();
+    ifSWithdrawAtStart();
+    overDrawSavings();
+
+
+    $("#savings .input").val("");
+  })
+
+// savings functions
+
```

Comment:
You have a lot of functions here. I think you could tighten up your code a lot if you were to combine some of these. For example, functions that capture all the logic behind checkingDeposit, checkingWithdraw, savingsDeposit and savingsWithdraw.